### DISCUSSION TITLE
Filtrare vat_types per updated_at

### DISCUSSION THREAD

**Discussion by ibreschi** [fattureincloud/api#464] [Q&A] [answered]
Con l'endpoint `GET /c/<id>/info/vat_types` vorrei poter ritornare soltanto i vat_types che sono stati modificati dopo una certa data. 
Sapete se e' possibile utilizzare l'updated_at come per gli issued_documents? 

Molte grazie

**Comment by valmoz** [comment, marked as answer]
Buongiorno @ibreschi 
le confermo che il campo updated_at non esiste per i vat_types.

Grazie
Mauro

**Reply by ibreschi** [reply]
Grazie per la risposta @valmoz 

Sarebbe possibile aggiungerlo? Mi pare che questa funzionalità sia in linea con il resto dell API dove le altre risorse hanno la create, update date.

Cordialmente

**Reply by valmoz** [reply]
Buongiorno @ibreschi,
direi di no.

Infatti, contrariamente a quanto succede per altre entità come i Clients e gli Issued Document, che sono ovviamente pensati per essere creati e modificati spesso, i VatTypes sono delle configurazioni e in quanto tali pensate per essere modificate il meno possibile (spesso non è nemmeno necessario definirne di personalizzate, in quanto quelle rese disponibili di default dal nostro sistema sono sufficienti).

Grazie
Mauro
